.travis.yml

```diff
@@ -1,32 +1,32 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more contributor license
-# agreements. See the NOTICE file distributed with this work for additional information regarding
-# copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance with the License. You may obtain a
-# copy of the License at
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#      http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under
-# the License.
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# requiring sudo forces Travis to build in an environment with more RAM
+sudo: required
 language: java
-notifications:
-  irc:
-    channels:
-      - "chat.freenode.net#fluo"
-    use_notice: true
-    on_success: change
-    on_failure: always
-    skip_join: true
+# speed up builds; don't use cache with default install step
 cache:
   directories:
     - $HOME/.m2
 install: echo NOOP Skipping pre-fetch of Maven dependencies
 jdk:
   - openjdk8
+before_script:
+  - unset _JAVA_OPTIONS
```

Comment:
A comment on why this is unset would be nice.

Reply:
Not sure how to make it brief enough to be useful inline, and I'm not 100% confident it is actually necessary. I think the git blame line is sufficient for now. If it becomes causes a lot of confusion in future, I'll spend more time trying to find the right wording for an inline comment.